### PR TITLE
cleanup: update metadata.json license

### PR DESCRIPTION
**Pull request by b4ldr**
This PR rupdates the licence in the metadata.json file to match the LICENCE file in the repo

@dalen can you just confirm this PR, i wouldn't want to accidentally change your licencing intentions, thanks

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "3.0.0",
   "author": "Erik Dalen <erik.gustav.dalen@gmail.com>",
   "summary": "Query functions for DNS",
-  "license": "Expat License",
+  "license": "MIT",
```
Comment (ekohl) [inline metadata.json:6]: Is this really the same license?

**Review by ekohl — COMMENTED** [review]
Normally we'd include the dropping of the lock file in https://github.com/voxpupuli/puppet-dnsquery/pull/117, since our modules require that.

File: metadata.json (hunk 1 above)
Comment (b4ldr) [reply metadata.json:6]: https://github.com/voxpupuli/puppet-dnsquery/blob/master/LICENSE is the [MIT licence text](https://opensource.org/licenses/MIT)

**Comment by ekohl** [comment]
I'd also suggest to disable Rubocop initially and fix it up later. We have https://github.com/voxpupuli/gha-puppet#basic-tests, but don't expose this in our modulesync config. I'd just patch it up manually now.

**Review by bastelfreak — APPROVED** [review]

**Comment by b4ldr** [comment]
@dalen awesome thanks

**Pull request merged**